tools/integration_tests/util/operations/file_operations.go

```diff
@@ -141,3 +143,45 @@ func CloseFile(file *os.File) {
 		log.Printf("error in closing: %v", err)
 	}
 }
+
+func ReadFileSequentially(filePath string, numberOfBytes int64) (content []byte, err error) {
+	content = make([]byte, numberOfBytes)
+	chunk := make([]byte, NumberOfBytesReadFromFile)
+	var offset int64 = 0
+
+	file, err := os.OpenFile(filePath, os.O_RDWR|syscall.O_DIRECT, FilePermission_0600)
```

Comment:
as discussed earlier, we need to close the file. Also, we can open the file with read only flag also ?